### DISCUSSION TITLE
Dynamic Schema change

### DISCUSSION THREAD

**Discussion by AhmadDarwich** [jOOQ/jOOQ#16701] [Q&A]
Hey :)
I’m trying to use clickhouse with jooq. My l table could change dynamically at runtime, is there a possible way to update the generated classes dynamically with jooq preventing a new release? 
thank you!

**Comment by lukaseder** [comment]
See: https://www.jooq.org/doc/latest/manual/sql-building/dsl-context/custom-settings/settings-render-mapping/
